### PR TITLE
fix(select): select design updated.

### PR DESCRIPTION
**Pull request by creador-dev**

### CODE REVIEW

File: packages/ui/select/src/select-base.tsx [hunk 1]
```diff
@@ -200,7 +200,11 @@ const Select: React.FC<SelectBaseProps> = ({
 		},
 		...(!isSearchable && {
 			dropdownToggle: () => {
-				dropdownRef.current?.open()
+				if (isDropdownOpen) {
+					dropdownRef.current?.close()
```
Comment (emgk) [inline packages/ui/select/src/select-base.tsx:204]: @creador-dev here you can use .toggle() method

File: packages/ui/dropdown/src/dropdown-menu-item.tsx [hunk 2]
```diff
@@ -59,19 +65,68 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
...
+			case "select":
+				break
+			case "select-checkbox":
+				content = (
+					<Checkbox
+						label={content}
+						isSmall={true}
+						isChecked={isChecked}
+						isDisabled={isDisabled}
+						{..._renderHTMLPropsSafely(_checkboxProps)}
```
Comment (emgk) [inline packages/ui/dropdown/src/dropdown-menu-item.tsx:81]: do not wrap _checkboxProp with _renderHTMLPropsSafely, pass them directly

File: packages/ui/dropdown/src/dropdown-menu-item.tsx [hunk 3]
```diff
@@ -21,8 +22,13 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
 	isDisabled,
 	onClick,
 	variation = "",
+	isChecked = false,
```
Comment (emgk) [inline packages/ui/dropdown/src/dropdown-menu-item.tsx:25]: now as we have `_checkboxProps` prop specifically, we can pass isChecked with this, so you can remove this

File: packages/ui/dropdown/src/dropdown-menu-item.tsx [hunk 4]
```diff
@@ -59,19 +65,68 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
...
+						isChecked={isChecked}
+						isDisabled={isDisabled}
+						{..._renderHTMLPropsSafely(_checkboxProps)}
+					/>
+				)
+				break
+			case "select-variable":
+				IconTag = IconTag ? IconTag : Icons.Add
+				content = (
+					<>
```
Comment (emgk) [inline packages/ui/dropdown/src/dropdown-menu-item.tsx:88]: you can remove <></>

File: packages/ui/dropdown/src/dropdown-menu-item.tsx [hunk 5]
```diff
@@ -59,19 +65,68 @@ const DropdownMenuItem: FC<DropdownMenuItemProps> = ({
...
+						<div id={menuTitleId} className="sui-dropdown__menu-item--wrapper">
+							<div>
+								<IconTag size="sm" className="sui-dropdown__menu-item-icon" />
+							</div>
+							<div>
+								<span className="sui-dropdown__menu-item--title">
+									{children}
+								</span>
+								{variable && (
+									<span className="sui-dropdown__menu-item--var">{` {${variable}}`}</span>
```
Comment (emgk) [inline packages/ui/dropdown/src/dropdown-menu-item.tsx:98]: Make the variable param accept React.ReactNode values and remove curly braces, so if a dev wants to pass any custom DOM, they can.

**Comment by creador-dev** [comment]
Attention! There are failing test cases that need to be fixed.

**Pull request closed without merging**